tests/enum/tenum.nim

```diff
@@ -128,22 +128,21 @@ block tnamedfields:
   doAssert $x == "abc", $x
 
 
-block toptions:
+block tfakeOptions:
```

Comment:
unrelated change (for convenience) to avoid getting false positive hits when searching for enum members

Reply:
=> moved to https://github.com/nim-lang/Nim/pull/18291